tests/dashbio_demos/app_manhattan_plot.py

```diff
@@ -83,12 +91,12 @@ def layout():
                             id='mhp-slider-genome',
                             vertical=False,
                             updatemode='mouseup',
-                            max=9,
+                            max=4,
```

Comment:
Does `4` happen to be `DATASET.shape[1]` or something?

Reply:
No, it's based on the y-values of the data (9 was too high -- there were no points at that value).

Reply:
I had to dig up the component and the data to figure out this y-value must be `'ZSCORE'`. So, it wouldn't hurt to take the max and then ceiling of that variable (plus some padding if desirable)... so the {reader, reviewer, reuser} of this piece of code has some idea of what is going on.